src/Core/Feature.js

```diff
@@ -223,6 +229,15 @@ class Feature {
     get geometryCount() {
         return this.geometries.length;
     }
+
+    setParentStyle(parentStyle) {
+        if (this.style && this.style.isStyle) {
+            this.style.parent = parentStyle;
```

Comment:
may be add inheritance when you instance  Geometry?

Reply:
I didn't, because `Feature.style` is always present, and we don't always want a style on a geometry (see vector tiles). However, I set the inheritance right after instancing it in `GeoJsonParser`.